airflow/bin/cli.py

```diff
@@ -776,10 +795,10 @@ class CLIFactory(object):
             default=False),
         # scheduler
         'dag_id_opt': Arg(("-d", "--dag_id"), help="The id of the dag to run"),
-        'num_runs': Arg(
-            ("-n", "--num_runs"),
+        'run_duration': Arg(
```

Comment:
Commands that used to not fail will fail now, we'll need to put that in the header of the next release notes.